toqito/state_opt/tests/test_ppt_distinguishability.py

```diff
@@ -1,7 +1,7 @@
 """Test ppt_distinguishability."""
 import numpy as np
+import pytest
 
-from toqito.perms import swap_operator
```

Comment:
You could copy-paste the changes in #436 here as I combined most of the tests using `pytest.mark.parametrize`. 

Feel free to make any changes to the material you copy-paste. From what can see, you will have to make changes to ` dist_method_str, strategy_bool`. Or I can do this too. Let me know!

I'lll go ahead and close that PR. 

https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-github

Reply:
This one might be a bit more amenable to keeping things separate as each test is a bit specific. I'll keep as is for now, but if you disagree, I can open another issue as well of course.